### PR TITLE
Automate confirm address button, add mute-audio arg and add .gitignore exclusions

### PR DESCRIPTION
**Pull request by nicballarini**
This is to tidy up my call to confirm_address() in main().

### CODE REVIEW

**Comment by nicballarini** [comment]
Okay okay, I'm done mucking this place up. :)

File: Amz.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     os.mkdir(cookies_full_path)
 # chromeOptions.add_argument('--user-data-dir=chrome-data')         # Linux way
 chromeOptions.add_argument('--user-data-dir=' + cookies_full_path)  # Windows way (full path)
+chromeOptions.add_argument('--mute-audio')
```
Comment (Penlo) [inline Amz.py]: Good addition!

File: Amz.py [hunk 2]
```diff
@@ -103,6 +104,19 @@ def give_away_type(driver):
...
 
+def confirm_address(driver):
+    # This will click the "confirm address" button for you
+    try:
+        driver.find_element(By.XPATH, '//input[@name="ShipMyPrize"]')
+        submit = WebDriverWait(driver, 20).until(
+			EC.element_to_be_clickable(
+				(By.XPATH, '//input[@name="ShipMyPrize"]'))
+		)
+        submit.click()
```
Comment (Penlo) [inline Amz.py]: You need to add `return True` here

File: Amz.py [hunk 3]
```diff
@@ -222,8 +236,13 @@ def main():
...
                 print('You won!')
-                you_won = True
-                break
+                # Call the confirm_address function
+                if confirm_address(driver):
+                    # If you win, will continue; exit if issue occurred.
+                    time.sleep(15)
+                    continue
+                else:
+                    break
```
Comment (Penlo) [inline Amz.py]: replace `break` with `return True` here

File: Amz.py (hunk 2 above)
Comment (nicballarini) [reply Amz.py]: fixed in next commit: https://github.com/Penlo/Amazon-Giveaway-Bot/pull/12/commits/856a575e6d8d20a617488f7e5daf721838e571f4

File: Amz.py (hunk 3 above)
Comment (nicballarini) [reply Amz.py]: fixed in next commit: https://github.com/Penlo/Amazon-Giveaway-Bot/pull/12/commits/856a575e6d8d20a617488f7e5daf721838e571f4

**Pull request merged**